documentation/Known-SDK-Driver-Compatibility/Overview.md

```diff
@@ -1,6 +1,8 @@
 # Abstract
 This folder contains, for known SDK drivers such as Windows Performance Analyzer, a mapping between SDK versions and all versions of the driver that can load plugins compiled against that SDK version.
 
+When you are creating an SDK plugin, you should refer to the compatability lists for the programs you want your plugin to work for. For example, if you intend your plugin to be used only with Windows Performance Analyzer, you will want to reference [its compatability list](./WPA.md).
```

Comment:
I intentionally left "programs" vague because it may or may not be a UI/SDK driver. For example I could create a wrapper program around the engine that has no UI, and my plugin could be loaded there.